client/express-checkout/index.js

```diff
@@ -192,32 +196,50 @@ jQuery( ( $ ) => {
 
 			wcpayECE.showButton( eceButton );
 
-			wcpayECE.attachButtonEventListeners( eceButton );
+			eceButton.on( 'click', ( event ) => {
+				// TODO: handle cases where we need login confirmation.
 
-			eceButton.on( 'click', function ( event ) {
-				const clickOptions = {
-					business: {
-						name: 'Mikes Bikes',
-					},
-					lineItems: [
-						{ name: 'Bike', amount: 200 },
-						{ name: 'Helmet', amount: 300 },
-					],
-					shippingAddressRequired: true,
-					shippingRates: [
-						{
-							id: '1',
-							amount: 500,
-							displayName: 'Standard Shipping',
-						},
-						{
-							id: '2',
-							amount: 1000,
-							displayName: 'Expedited Shipping',
-						},
-					],
+				// TODO: This is not ideal but should work and it's how it's implemented right now for PRBs.
+				wcpayECE.addToCart();
```

Comment:
This may need to be product page specific? I don't think this is needed for the cart and checkout pages which use the same function, so wrapping this in an `if` statement is probably needed.

Reply:
Yes. This PR is still WIP. I was planning to test with the other pages where the ECE can be enabled.

Reply:
Fixed this in the merge.